components/pkg-export-kubernetes/src/lib.rs

```diff
@@ -34,13 +34,15 @@ use common::ui::UI;
 pub mod topology;
 pub mod error;
 pub mod manifest;
+pub mod manifestjson;
```

Comment:
A typo in the message of the commits that adds ManifestJson:
`s/repressentations/representations/`